modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Integrating software RAID arrays with {lvms}
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID array using the `mdadm` utility, and integrate the RAID array with {lvms}.
```

Comment:
```suggestion
You can create the redundant array of independent disk (RAID) array by using the `mdadm` utility, and integrate the RAID array with {lvms}. Logical Volume Manager (LVM) does not support creating a software RAID. 
```

Flipped the sentences based on potential internet search truncation. Best to start with what a user can do and then mention the limitation.

Reply:
Updated as per the suggestion.

Reply:
:heavy_check_mark: